package.json

```diff
@@ -1,16 +1,44 @@
-{
-  "name": "task1",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "react": "^16.3.1",
-    "react-dom": "^16.3.1",
-    "react-scripts": "1.1.4"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
-  }
-}
\ No newline at end of file
+{
+    "name": "react_task2",

```

Comment:
Keep a unique name in package.json, to make clear for what tasks this changes are, is better keep it in the branch name.

Reply:
Okay